### ISSUE TITLE
Not all animations are being removed.

### ISSUE THREAD

**Issue by fhuewhd** [rauenzi/BetterDiscordApp#228]
I have a sensory disorder.  The animations that are part of Discord's interface are very uncomfortable and distracting.  I use BetterDiscord to get rid of them and make the program usable.  My Custom CSS screen reads:

```
*{
    animation:none !important;
    transition:none !important;
}
```

(I don't speak CSS myself - somebody told me to put this in).

I also have Minimal Mode enabled.

Recently, however, there are at least three very problematic ones that it's not blocking, which I'm sure never used to appear since I've been using BetterDiscord until recently:

- The three prancing dots next to a person's name in the list of people in the server when they're typing, when you have a channel displayed.
- Ditto next to a person's name in the list of private conversations, when you have a private conversation displayed.
- The fruit machine-like animation when somebody puts a "reaction" on a post, or at least when more than one person does in quick succession.

There's also the way that dialogue boxes "swoosh" into view when you press, for example, "Change Nickname", or try to delete a message.  But that's been going on for a while, and I know to expect it and cover my eyes before doing these things.  But it would still be nice to have it gone.

How can this be fixed?  Is it a flaw in BetterDiscord (if so, _please_ fix it!), or is there something I can add to my Custom CSS or something else to remove these animations?  A way to get rid of these animations, and as soon as possible, would be VERY much appreciated!

**Comment by qwerasd205** [comment]
To start off, it is not a flaw in BetterDiscord.
The reason these animations aren't being blocked by the CSS you have is because they're unfortunately JavaScript powered. It _is_ possible for most of these to be prevented with CSS, however -- it would just take a little bit of work. I think it would be a good idea for someone to make a reduced/no motion theme for people who have similar issues to you. I would make one personally, but I don't really have time to do it at the moment. I will actually talk to some theme devs to see if they're interested in creating it since it's a shame that using Discord is uncomfortable for you.

**Comment by rauenzi** [comment]
Q gave a good explanation above. I would encourage devs to look into creating a theme for this but as Q said above this isn't an issue with BD directly but I'm closing this.

My suggestion to you is that you suggest this in the theme/css channels in the support servers to get more visibility to devs (that is unless Q wants to take a stab at it).

**Comment by NFLD99** [comment]
im not sure that this covers everything but it will kill off the popup animations and typing 

```css
*{
  animation:none !important;
  transition:none !important;
}

circle[fill="white"]{
  opacity: 1!important;
}

.modal-3c3bKg[style*="transform: scale"],.popout-2iWAc-,.animating-rRxada{
  transform:none !important;
  opacity: 1!important;
}

.backdrop-1wrmKB{
  opacity: 1!important;
}
```

**Comment by Vaxuuu** [comment]
Hey! I decided to try and work on it and managed to do some things, starting off with the reactions. The number will look wonky for a split second when it changes but it should be fine otherwise. I got to remove the animation from the dots in the user list. However, the animation of the status indicator expanding is still there (blame my stupidity; I just couldn't figure it out). The pop-outs that "swoosh" into the screen also were rid off their animation. 

When it comes to the dots in private chats, I just couldn't recreate them but I hope their class is the same. I have high hopes I made your Discord experience a tiny bit better.

As of writing this NFLD has already posted his work (heck >:C) but I'll post this anyway. 
```
/*reactions*/
#app-mount .reaction-1ELvT8.reactionMe-23mbRf /*reacting to stuff*/{
    transform: none !important;
    opacity: 1 !important;
}
.reaction-1ELvT8.reactionMe-23mbRf .reactionCount-2ddRoS div[style*="position: absolute;"], .reactionCount-2ddRoS div[style*="position: absolute;"] {
    transform: none !important; /*reaction number - will look wonky for a split second*/
}
/*those dot things*/
#app-mount .dots-3Bkt3k circle[fill="white"] {
    opacity: 1 !important;
    r: 2;
}
/*popouts and the backdrop*/
#app-mount .inner-1ilYF7, .modal-3c3bKg /*popouts*/{
    transform: none !important;
    opacity: 1 !important;
}
#app-mount .backdrop-1wrmKB, .backdropWithLayer-3_uhz4 /*backdrop (background)*/ {
    transform: none !important;
    opacity: 0.8 !important;
}
```

**Comment by fhuewhd** [comment]
@Vaxuuu You nailed it!  Thank you so much!  Since that worked I haven't tried out @NFLD99 's version, but I expect that works just as well, all the same things seem to be in it as far as I can judge.  

Never mind about the status indicator expanding if neither of you can figure it out, it would be nice, and tidier-looking too, to have it gone but that doesn't bother me so long as it at least only does that and doesn't proceed to shimmer at me the whole time.  

What we should really do is make this a theme, or part of a theme, so it could be made generally available from the page (part of the wiki, I think, possibly the wiki for the original Jiiks version) where themes are published.  Then anyone who wants it can find it.  I have no idea how to do that, though, so I'll leave that to you!

**Comment by rauenzi** [comment]
> possibly the wiki for the original Jiiks version

That wiki is not where themes are published though, they are published on our Discord servers. That wiki is very old and outdated.

**Comment by fhuewhd** [comment]
Perhaps there should be some sort of thing posted for people who actually wouldn't be able to use the Discord servers for long enough to find it without some such thing.  I mean, I had to ask about it on here, in an "issue" thread which is presumably long out of sight by now, to find out only how to do the "animations none transitions none" code.  Couldn't have done it without.

**Comment by rauenzi** [comment]
I'm not sure exactly what you're asking for can you elaborate. The servers are quite well known and they are advertised both on this repository and the original Jiiks one.

**Comment by qwerasd205** [comment]
I'm pretty sure that they are saying that struggled to navigate the servers before getting the fixes they need because of their sensory issues.

**Comment by fhuewhd** [comment]
Yes, exactly.  It's no use somebody who is unable to use Discord having to use Discord to get the add-on to enable them to use Discord.
